src/content/structured/components/text-input/code.mdx

```diff
@@ -325,6 +325,53 @@ export const snippetsValidation = [
   </div>
 </ComponentPreview>
 
+### With min and max values
+
+In this example, an error message will display when the input is either below the min value or above the max value. This only affects numeric input fields.
+
+export const snippetsMinMax = [
+  {
+    language: "Web component",
+    snippet: `<ic-text-field
+  validation-status="error"
+  validation-text="now it has really gone to (coffee) pot"
+  label="How many coffees do you want?"
+  helper-text="Our coffee machines have a maximum capacity of 4 coffees at a time."
+  type="number"
+  inputmode="numeric"
+  min="1"
+  max="4"
+>
+</ic-text-field>`,
+  },
+  {
+    language: "React",
+    snippet: `<IcTextField
+  validationStatus="error"
+  validationText="now it has really gone to (coffee) pot"
```

Comment:
I think these validations don't show what we want them to with min/max, I think it'd be more setting the value to say 6 (over the max value) and showing that validation?

Reply:
I think only the web component was updated here. But also, unless I'm getting the wrong end of the stick, I think the validation needs to just be removed? If you have the value set to 6 in this example, the validation is automatically generated. Which I think is what we want to display here?